# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../aqbanking/uic-includehints.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- aqbanking-1.6.0beta/src/plugins/backends/aqdtaus/ui/qt3/libs/views/accountview.ui.orig	2005-02-24 05:07:08.000000000 +0100
+++ aqbanking-1.6.0beta/src/plugins/backends/aqdtaus/ui/qt3/libs/views/accountview.ui	2005-11-02 11:19:53.000000000 +0100
@@ -149,7 +149,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>accountlistview.h</includehint>
-</includehints>
 </UI>
--- aqbanking-1.6.0beta/src/plugins/backends/aqgeldkarte/ui/qt3/libs/views/accountview.ui.orig	2005-08-11 14:45:37.000000000 +0200
+++ aqbanking-1.6.0beta/src/plugins/backends/aqgeldkarte/ui/qt3/libs/views/accountview.ui	2005-11-02 13:07:31.000000000 +0100
@@ -149,7 +149,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>accountlistview.h</includehint>
-</includehints>
 </UI>
--- aqbanking-1.6.0beta/src/plugins/backends/aqofxconnect/ui/qt3/libs/views/accountview.ui.orig	2005-05-14 00:11:00.000000000 +0200
+++ aqbanking-1.6.0beta/src/plugins/backends/aqofxconnect/ui/qt3/libs/views/accountview.ui	2005-11-02 13:10:15.000000000 +0100
@@ -130,7 +130,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>accountlistview.h</includehint>
-</includehints>
 </UI>
--- aqbanking-1.6.0beta/src/plugins/backends/aqofxconnect/ui/qt3/libs/views/userview.ui.orig	2005-05-14 00:11:00.000000000 +0200
+++ aqbanking-1.6.0beta/src/plugins/backends/aqofxconnect/ui/qt3/libs/views/userview.ui	2005-11-02 17:04:04.000000000 +0100
@@ -163,7 +163,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>userlistview.h</includehint>
-</includehints>
 </UI>
